src/common/ViewFileButton.tsx

```diff
@@ -51,38 +52,15 @@ export default function ViewFileButton(props: ViewFileProps) {
     )
 }
 
-export function DownloadFileButton(props: ViewFileProps) {
-    const { axiosFactory } = useLogionChain();
-    if (axiosFactory === undefined) {
-        return null;
-    }
-    return (
-        <Button onClick={ () => openFile(axiosFactory(props.nodeOwner)!, props) } className="ViewFileButton">
-            { props.children }
-        </Button>
-    )
-}
-
 export interface DownloadFilesProps {
     nodeOwner: string;
     files: FileInfo[];
-    children?: Children;
+    axiosFactory: AxiosFactory
 }
 
-export function DownloadFilesButton(props: DownloadFilesProps) {
-    const { axiosFactory } = useLogionChain();
-    if (axiosFactory === undefined) {
-        return null;
+export async function openFiles(props: DownloadFilesProps) {
```

Comment:
So this is not a React component anymore? In that case, I would rename props into something else (args, params, ...) to prevent any confusion. Also, the file should be renamed as well as the `DownloadFilesProps` interface.

Reply:
`DownloadFileButton` and `DownloadFilesButton` were already present, but not used. Unfortunately this kind of all-in-one button does not integrate verry well with Dialog attribute `actions`. I just got rid of unused stuff... Will rename as requested.